src/client/contextChannels.ts

```diff
@@ -391,10 +412,10 @@ export const DEFAULT_CHANNEL_ID: ChannelId = 'default';
 /**
  * The channel in which all windows will initially be placed.
  *
- * All windows will belong to exactly one channel at all times. If they have not explicitly
- * been placed into a channel via a {@link ChannelBase.join} call, they will be in this channel.
+ * All windows will belong to exactly one channel at all times. If they have not explicitly been placed into a channel
+ * via a {@link ChannelBase.join} call, they will be in this channel.
  *
- * If an app wishes to leave a system channel it can do so by (re-)joining this channel.
+ * If an app wishes to leave a system/app channel it can do so by (re-)joining this channel.
```

Comment:
> If an app wishes to leave a system/app channel

> If an app wishes to leave a channel

Reply:
I put it as "system/app channel" rather than just" channel" as otherwise it's inaccurate if you're already in the default channel.

Would "If an app wishes to leave any other channel, it can do so by (re-)joining this channel." be an improvement?

Reply:
Yeah that sounds better.

Reply:
✅